bridge/bindings/qjs/script_wrappable.cc

```diff
@@ -263,6 +265,15 @@ void ScriptWrappable::InitializeQuickJSObject() {
   jsObject_ = JS_NewObjectClass(ctx_, wrapper_type_info->classId);
   JS_SetOpaque(jsObject_, this);
 
+  // Set .constructor to the constructor function.
+  JSValue constructor = context_->contextData()->constructorForType(wrapper_type_info);
+  JS_DefinePropertyValue(ctx_, jsObject_, built_in_string::kconstructor.Impl(), JS_DupValue(ctx_, constructor),
+                         JS_PROP_C_W_E);
```

Comment:
The `constructor` property should be defined on the constructor's prototype instead of the instance.
And should not be enumerable.
Following the spec https://tc39.es/ecma262/#sec-makeconstructor.

Reply:
fixed